tests/Fixtures/library/Model/Collection.php

```diff
@@ -6,7 +6,7 @@ class Collection implements \ArrayAccess, \Countable, \IteratorAggregate
 {
     private $models = [];
 
-    public function __construct(array $models)
+    public function __construct(array $models, $strTable)
```

Comment:
I doubt that this is correct?

Reply:
It is. But that's also a dummy that should be removed completely.